src/helper-functions/urls.js

```diff
@@ -1,13 +1,22 @@
 const baseURL = `https://api.realdevsquad.com`;
 const imgBaseURL = `https://raw.githubusercontent.com/Real-Dev-Squad/website-static/main`;
 const getMembersURL = `${baseURL}/members`;
+const cloudinaryImageUrl = `https://res.cloudinary.com/realdevsquad/image/upload`;
```

Comment:
Can we please move this also to constants?

Reply:
This file contains all the url's, I don't get the reason for moving this to another constant file and using it here again

Reply:
Ah! sorry, my bad.